### PR TITLE
fix : updated box shadow color

### PR DESCRIPTION
**Pull request by prankurpandeyy**

### CODE REVIEW

**Comment by Naman-Saxena1** [comment]
Looks fine.

**Comment by rahulk31** [comment]
Looks good, can be merged.

File: components/Cards/cards.css [hunk 1]
```diff
@@ -112,5 +112,5 @@ button {
 
 /* shadow class */
 .card-with-shadow {
-    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+    box-shadow: rgba(206, 111, 111, 0.35) 0px 5px 15px;
```
Comment (varun1233) [inline components/Cards/cards.css]: use rem

**Review by sri-rishi — APPROVED** [review]
Looks fine to me for merging

**Pull request merged**